test/e2e/e2e_test.go

```diff
@@ -18,11 +18,16 @@ package e2e
 
 import (
 	"testing"
+	"flag"
 
 	"github.com/kubernetes-incubator/service-catalog/test/e2e/framework"
 )
 
+var brokerImage string
```

Comment:
can we call this broker image flag so if all I'm seeing later is in util.go, I know that it was probably a flag passed in?